README.md

```diff
@@ -111,16 +147,14 @@ vars:
   stripe__payout_metadata:
     - name: 123
       alias: one_two_three
-  stripe__plan_metadata:
-    - name: rename
-    - alias: renamed_field
+  stripe__price_plan_metadata: ## Used for both Price and Plan sources
```

Comment:
My apologies for not catching this, but we will need to update the CHANGELOG that this variable was renamed. Actually, it looks like we didn't include any of the metadata variables in the previous -> new variable names. Can you add those there.

Reply:
@fivetran-joemarkiewicz  it looks like the other metadata variables were already prefixed with `stripe`, so I don't think they were renamed?

Reply:
Ah good catch. Thanks for calling that out. If that is the case then we can just call out the price_plan change.

Reply:
Yup -- added that 
- `stripe__plan_metadata` variable has been renamed to `stripe__price_plan_metadata`